uint8 TABLE=0
uint8 CHAIR=1
uint8 PERSON=2
uint8 PERSON_HEAD=3
uint8 MILK=4
